sharding/vmc.go

```diff
@@ -66,3 +66,42 @@ func initVMC(c *Client) error {
 
 	return nil
 }
+
+// initVMCValidator checks if the account is a validator in the VMC. If
+// the account is not in the set, it will deposit 100ETH into contract.
+func initVMCValidator(c *Client) error {
+
+	// TODO: Check if account is already in validator set. Fetch this From
+	// the VMC contract's validator set
+
+	// Unlocks the current account from the keystore
+	accounts := c.keystore.Accounts()
+	if len(accounts) == 0 {
+		return fmt.Errorf("no accounts found")
+	}
+
+	if err := c.unlockAccount(accounts[0]); err != nil {
+		return fmt.Errorf("unable to unlock account 0: %v", err)
+	}
+
+	// Deposits 100ETH into the VMC from the current account
+	ops := bind.TransactOpts{
+		From:  accounts[0].Address,
+		Value: depositSize,
+		Signer: func(signer types.Signer, addr common.Address, tx *types.Transaction) (*types.Transaction, error) {
```

Comment:
Can we abstract this signer func so that it can be reused? we are already using it elsewhere.